website/addons/s3/static/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bucketLocations": {
+        "": "US Standard",
+        "EU": "Europe Standard",
+        "us-west-1": "California",
+        "us-west-2": "Oregon",
+        "ap-northeast-1": "Tokyo",
+        "ap-southeast-1": "Singapore",
+        "ap-southeast-2": "Sydney",
+        "cn-north-1": "Beijing"
```

Comment:
Beijing is currently not supported with boto see: https://github.com/jschneier/django-storages/issues/28
Couldn't [this](http://docs.pythonboto.org/en/latest/ref/s3.html#boto.s3.regions) be used to generate a "bucketlist"